lambdas/services/base/dynamo_service.py

```diff
@@ -122,6 +122,14 @@ def update_item(self, table_name: str, key: str, updated_fields: dict):
             ExpressionAttributeValues=expression_attribute_values,
         )
 
+    def remove_attribute_from_item(self, table_name: str, key: str, attribute: str):
+        table = self.get_table(table_name)
+
+        table.update_item(
+            Key={"ID": key},
+            AttributeUpdates={attribute: {"Action": "DELETE"}},
```

Comment:
UpdateExpression preferred in boto3, as AttributeUpdates is considered deprecated.

```
        table.update_item(
            Key={"ID": key},
            UpdateExpression=f"REMOVE {attribute}"
        )
```